cpp/src/io/orc/reader_impl.cu

```diff
@@ -861,13 +961,9 @@ column_buffer&& reader::impl::assemble_buffer(const int32_t orc_col_id,
   col_buffer.name = _metadata->get_column_name(0, orc_col_id);
   switch (col_buffer.type.id()) {
```

Comment:
maybe an if is sufficient here?

Reply:
was thinking  I might requier this for MAP, but will take care at that time